File: C++/Check if a number is divisible by 8.cpp
 class Solution{
    public:
    int DivisibleByEight(string s){
        int n = s.size();
        if(n>3){
            s = s.substr(n-3,3);
        }
        if(stoi(s)%8==0){
            return 1;
        }
        else{
            return -1;
        }
    }
};

File: JAVA/Check if a number is divisible by 8.java
class Solution{
    int DivisibleByEight(String s){
        if(s.length() == 0) return -1;
        int num = 0;
        num = Integer.parseInt(s.substring(Math.max(0, s.length() - 3)));
        if(num % 8 == 0) return 1;
        return -1;
    }
}
